vm/state/src/DefaultTevmStateManager.ts

```diff
@@ -0,0 +1,17 @@
+import type { TevmStateManagerInterface } from './TevmStateManager.js'
+import { DefaultStateManager } from '@ethereumjs/statemanager'
+
+export class DefaultTevmStateManager
+	extends DefaultStateManager
+	implements TevmStateManagerInterface
+{
+	getAccountAddresses = () => {
```

Comment:
Instead of doing this inheritance twice you can just make this a utility function that you export from state package

```typescript
export const getAccountAddresses = (state: StateManager): Array<Address> => {
  return this._accountCache?._orderedMapCache?.map(([address]) => address)
}
```

That allows you to delete this class, delete the TevmEVM class etc.